internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```diff
@@ -305,4 +312,10 @@ EnvironmentManagerRole:
             - 'cloudformation:DescribeStacks'
             - 'cloudformation:DeleteStack'
           Resource:
-            - !Sub 'arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${AWS::StackName}/*'
\ No newline at end of file
+            - !Sub 'arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${AWS::StackName}/*'
+        - Sid: RDS
+          Effect: Allow
+          Action:
+            - 'rds:DescribeDBInstances'
+            - 'rds:DescribeDBClusters'
+          Resource: "*"
```

Comment:
do you think we should/can scope this down?

Reply:
it doesn't support tag conditions 😢